.github/workflows/solvcon_install.yml

```diff
@@ -1,4 +1,4 @@
-name: solvcon_install
+name: Build with Conda
```

Comment:
Aligned with the naming convention of `devenv` style so make us easier to maintain the CI.

Reply:
I prefer to not change the names because it will create new items in the column of workflows and mislead:

<img width="574" alt="image" src="https://user-images.githubusercontent.com/399122/152450837-017946c5-e9d7-49ac-97e6-88b95d085317.png">

Reply:
To align with `devenv`, a line of comment should work too.

Reply:
Agree.

FWIW, we can delete multiple old runs in a batch via Github API. Reference https://stackoverflow.com/questions/57927115/anyone-know-a-way-to-delete-a-workflow-from-github-actions

In the short run, I will update this pull request with the old names until we are happy to use the new naming style (and remove the old runs via Github API).

Reply:
I did that in other repository and if I am not mistaken, the UI only allows to delete one run a time.  There are 424 runs in the workflow and clicking 424 times seems to be a big waste of time.

In addition, "the old runs" serve as good reference.  I do not see benefits of deleting them.  I do not see benefits of using a new workflow name either.

Aligning with devenv workflow name does not seem to bring us benefits as long as we add a line of comment summarizing the workflow.